accepted/2025/cve-schema/cve-schema.md

```diff
@@ -0,0 +1,1047 @@
+# Publishing CVE information as structured data
+
+We publish vulnerability fixes and matching CVE documentation nearly every month. These are primarily [documented on GitHub in HTML](https://github.com/dotnet/announcements/issues?q=is%3Aissue%20state%3Aopen%20label%3ASecurity) for people to read. It is well understood that HTML is a presentation format and not appropriate for structured processing. We intend to offer CVE information in structured JSON format (and likely markdown, too).
```

Comment:
Forgive me if I'm missing something, but isn't the *primary* place CVEs are documented in the primary source that the Microsoft CNA provides to NVD?

E.g. https://nvd.nist.gov/vuln/detail/cve-2024-38081#range-16178023 provides https://msrc.microsoft.com/update-guide/vulnerability/CVE-2024-38081 as the primary source.

There is no link to GitHub, and I rarely come across vulnerability aggregators referencing the github issues: They're not linked to from the CVE record, so why would they be trusted?

Automated tools use NVD as the source of truth for CVEs. e.g. https://security.snyk.io/vuln/SNYK-ALPINE320-DOTNET6BUILD-7448438

Reply:
```suggestion
We publish vulnerability fixes and matching CVE documentation nearly every month. These are primarily [documented in HTML](https://github.com/dotnet/announcements/issues?q=is%3Aissue%20state%3Aopen%20label%3ASecurity) for people to read. It is well understood that HTML is a presentation format and not appropriate for structured processing. We intend to offer CVE information in structured JSON format (and likely markdown, too).
```